src/serializers/error_serializer.cr

```diff
@@ -0,0 +1,14 @@
+# This is the default error serializer generated by Lucky.
+# Feel free to customize it in any way you like.
+class ErrorSerializer < BaseSerializer
+  def initialize(
+    @message : String,
+    @details : String? = nil,
+    @param : String? = nil # If there was a problem with a specific param
```

Comment:
This comment didn't make sense to me for whatever reason until I read it a few times. What do you think about:

```suggestion
    @param : String? = nil # so you can track which param (if any) caused the problem
```

Reply:
This is the comment that's generated in apps by default https://github.com/luckyframework/lucky_cli/blob/master/src/web_app_skeleton/src/serializers/error_serializer.cr#L7

I'm all down for clarity, but if we do decide to change it, then we should make the same change in the CLI.

Reply:
I like the change and I'll change it in CLI too. Thanks both for pointing those things out